derived_images/mongodb/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM opensuse:42.1
+MAINTAINER Evgeny Shmarnev "eshmarnev@suse.com" \
+	   Paul Gonin "pgonin@opensuse.org"
+ENV REFRESHED_AT 2016-14-07
+
+RUN set -x \
+&&  zypper ar http://download.opensuse.org/repositories/server:/database/openSUSE_Leap_42.1/server:database.repo \
+&&  zypper -n --gpg-auto-import-keys refresh \
```

Comment:
Please don't automatically import all the repositories keys, this is insecure. You just need to make sure the key of the `server:database` project is imported before running `zypper`.

Take a look at what is done [here](https://github.com/openSUSE/docker-containers/blob/master/derived_images/portus/docker/Dockerfile#L8).

The fingerprint of the `server:database` is `116EB86331583E47E63CDF4D562111AC05905EA8`.

In case you wonder about how to obtain this magic value:

1. download [this key](http://download.opensuse.org/repositories/server:/database/openSUSE_Leap_42.1/repodata/repomd.xml.key)
2. import it into your gpg keychain
3. use `gpg --fingerprint 05905EA8`